pkgs/development/python-modules/acstore/default.nix

```
@@ -0,0 +1,21 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  pname = "acstore";
+
+  version = "20230226";
```

The latest release would be `20240407`.

```suggestion
  pname = "acstore";
  version = "20230226";
  pyproject = true;
```

and add `build-system = [ setuptools ];` after `src`.